spinel/codec.py

```diff
@@ -843,6 +843,8 @@ def __init__(self, stream, nodeid, use_hdlc=FEATURE_USE_HDLC):
         self.stream = stream
         self.nodeid = nodeid
 
+        self.timeout = TIMEOUT_PROP
```

Comment:
could we provide a timeout parameter in the __init__ which default to be TIMEOUT_PROP?
thus helping to avoid setting the WpanApi's field member explicitly.
https://github.com/openthread/pyspinel/pull/71/commits/bd0a2dcc4b6f67f2642dc33c6f413500bc5ccd40#diff-3b4727a203ce0f96969d623334e33af9R89

Reply:
Updated.